### PR TITLE
feat: 🎸 add new VEW 3.1.0 modules

### PR DESCRIPTION
**Pull request by shuffledex**
add `templates`, `getSchedulesCountByTemplate`, `getAllBeneficiaries`
and `getAvailableTokens` methods

### CODE REVIEW

File: src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper/3.1.0.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+   */
+  public modifySchedule = async (params: ModifyScheduleParams): Promise<PolyResponse> => {
+    assert.assert(
+      await this.isCallerAllowed(params.txData, Perm.Admin),
+      ErrorCode.Unauthorized,
+      'Caller is not allowed',
+    );
+    this.checkSchedule(params.beneficiary);
+    assert.assert(params.startTime.getTime() >= Date.now(), ErrorCode.TooEarly, 'Date in the past');
+    // TODO: require(now < schedule.startTime, "Schedule started");
```
Comment (monitz87) [inline src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper/3.1.0.ts]: Why is this TODO here?

File: src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper/3.0.0.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+   */
+  public modifySchedule = async (params: ModifyScheduleParams): Promise<PolyResponse> => {
+    assert.assert(
+      await this.isCallerAllowed(params.txData, Perm.Admin),
+      ErrorCode.Unauthorized,
+      'Caller is not allowed',
+    );
+    this.checkSchedule(params.beneficiary);
+    assert.assert(params.startTime.getTime() > Date.now(), ErrorCode.TooEarly, 'Date in the past');
+    // TODO: require(now < schedule.startTime, "Schedule started");
```
Comment (monitz87) [inline src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper/3.0.0.ts]: Why is this TODO here?

File: src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper/3.1.0.ts (hunk 1 above)
Comment (shuffledex) [reply src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper/3.1.0.ts]: referent comment that I forgot to remove

File: src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper/3.0.0.ts (hunk 2 above)
Comment (shuffledex) [reply src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper/3.0.0.ts]: referent comment that I forgot to remove

File: src/index.ts [hunk 3]
```diff
@@ -349,4 +339,19 @@ export {
   Provider,
   Web3Wrapper,
   BigNumber,
+  VestingEscrowWalletContract_3_0_0,
```
Comment (monitz87) [inline src/index.ts]: You need to do the same for 3.1 as well

Comment (shuffledex) [reply src/index.ts]: I see... done!

**Pull request merged**